### ISSUE TITLE
EDSC-979: AST_L1A ordering changes breaking ability to order data for those who have ordered in the past

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#495]
It appears as though the AST_L1A form has changed. I see a broken page and console errors in ops when trying to order.

We need to verify that the form itself isn't broken and update our code to be robust against this sort of error.

This change needs a backport to UAT and ops.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

Approved by Katie and Dana via email

**Comment by earthdata-tasktop** [comment]
(Comment from Matt Martens):

I was able to filter my search down to 11 AST*L1A granules and successfully place an order in OPS.  I didn't have any problem with the AST*L1A form.  Patrick, can you reproduce the issue you saw?  If so, what type of order are you trying to place when you get this error?

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options.

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen.

There's nothing to be done on the LPDAAC side. We have a patch in the works to fix this on our end, and we'll be backporting to operations as soon as it's ready.

**Comment by earthdata-tasktop** [comment]
(Comment from Matt Martens):

[~pquinn] Please see previous comment.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

The fix has made it to ops and I've confirmed I can get past the configuration step now.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I am unable to reproduce the situation which causes the error, and Matt Martens was also not able to reproduce it either.  

Patrick wrote this ticket, and the error condition seems to be specific to his data access history.   I've talked to Patrick, and we don't know a way for me to try to reproduce this situation to verify the ticket.

Patrick wrote this comment about the error situation:  "EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options. 

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen." 

He also wrote a comment in the ticket which says "The fix has made it to ops and I've confirmed I can get past the configuration step now." 

Based on Patrick no longer seeing the error given his data access history, I am verifying the issue internally.

**Comment by earthdata-tasktop** [comment]
(Comment from Matt Martens):

[~pquinn] Please see previous comment.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

The fix has made it to ops and I've confirmed I can get past the configuration step now.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I am unable to reproduce the situation which causes the error, and Matt Martens was also not able to reproduce it either.  

Patrick wrote this ticket, and the error condition seems to be specific to his data access history.   I've talked to Patrick, and we don't know a way for me to try to reproduce this situation to verify the ticket.

Patrick wrote this comment about the error situation:  "EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options. 

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen." 

He also wrote a comment in the ticket which says "The fix has made it to ops and I've confirmed I can get past the configuration step now." 

Based on Patrick no longer seeing the error given his data access history, I am verifying the issue internally.

**Comment by earthdata-tasktop** [comment]
(Comment from Patrick Quinn):

The fix has made it to ops and I've confirmed I can get past the configuration step now.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I am unable to reproduce the situation which causes the error, and Matt Martens was also not able to reproduce it either.  

Patrick wrote this ticket, and the error condition seems to be specific to his data access history.   I've talked to Patrick, and we don't know a way for me to try to reproduce this situation to verify the ticket.

Patrick wrote this comment about the error situation:  "EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options. 

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen." 

He also wrote a comment in the ticket which says "The fix has made it to ops and I've confirmed I can get past the configuration step now." 

Based on Patrick no longer seeing the error given his data access history, I am verifying the issue internally.

**Comment by earthdata-tasktop** [comment]
(Comment from Dana Shum):

Migrating "Internal Verified" tickets to "Completed".

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I am unable to reproduce the situation which causes the error, and Matt Martens was also not able to reproduce it either.  

Patrick wrote this ticket, and the error condition seems to be specific to his data access history.   I've talked to Patrick, and we don't know a way for me to try to reproduce this situation to verify the ticket.

Patrick wrote this comment about the error situation:  "EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options. 

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen." 

He also wrote a comment in the ticket which says "The fix has made it to ops and I've confirmed I can get past the configuration step now." 

Based on Patrick no longer seeing the error given his data access history, I am verifying the issue internally.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

I am unable to reproduce the situation which causes the error, and Matt Martens was also not able to reproduce it either.  

Patrick wrote this ticket, and the error condition seems to be specific to his data access history.   I've talked to Patrick, and we don't know a way for me to try to reproduce this situation to verify the ticket.

Patrick wrote this comment about the error situation:  "EDSC saves your most-recently-applied access options and applies them the next time you choose to access data. In this case, I found that I had used the ESI service the last time I accessed data, and that wasn't available as an option anymore (I don't recall it ever being available, but that's beside the point). The page has errors when the previously-applied access options change in a way that breaks the available forms and/or there is no available form for the previously-applied options. 

If you don't see this error, you won't see this error. In my case, I see it every time, and it prevents me from getting past the config screen." 

He also wrote a comment in the ticket which says "The fix has made it to ops and I've confirmed I can get past the configuration step now." 

Based on Patrick no longer seeing the error given his data access history, I am verifying the issue internally.
